lib/platform.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const hashr = require('screwdriver-hashr');
+const BaseModel = require('./base');
+
+class PlatformModel extends BaseModel {
+    /**
+     * Construct a PlatformModel object
+     * @method constructor
+     * @param  {Object}    datastore         Object that will perform operations on the datastore
+     */
+    constructor(datastore) {
+        super(datastore);
+        this.table = 'platforms';
+    }
+
+    /**
+     * Create a platform
+     * @method create
+     * @param  {Object}   config                Config object to create the platform with
+     * @param  {String}   config.name           The name of the platform
+     * @param  {String}   config.version        The version of the platform
+     * @param  {Function} callback              fn(err)
+     */
+    create(config, callback) {
+        const id = hashr.sha1({
```

Comment:
We keep track of the of unique keys that should be used for hashing here: https://github.com/screwdriver-cd/data-schema/blob/master/model/platform.js#L97

Should we find a way to generically implement a hashing function for all models?